### DISCUSSION TITLE
Is there any way to do copy and paste between apps?

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#6970] [Help] [answered]
I am trying to merge two apps, by doing copy and paste pages selectively from one app to another.

Unfortunately, I see that the design editor does not support this. Is there a way to avoid manually reconstructing pages from scratch?

**Comment by melohagan** [comment, marked as answer]
Hi @dannyshin76 

It is currently no way to do this. It is a great idea though, and I've raised a feature request here: https://github.com/Budibase/budibase/issues/7136
